package_manifest.yaml

```diff
@@ -0,0 +1,415 @@
+---
+packages:
+  children:
+    foreman: {}
+    foreman-nonscl: {}
+    plugins: {}
+    plugins-nonscl: {}
+    katello: {}
+
+foreman:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    releasers:
+    - koji-foreman-nightly
+  hosts:
+    foreman: {}
+    rubygem-activerecord-session_store: {}
+    rubygem-addressable: {}
+    rubygem-algebrick:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-ancestry: {}
+    rubygem-apipie-bindings: {}
+    rubygem-apipie-params:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-apipie-rails: {}
+    rubygem-audited: {}
+    rubygem-autoparse: {}
+    rubygem-autoprefixer-rails: {}
+    rubygem-awesome_print: {}
+    rubygem-bootstrap-sass: {}
+    rubygem-bundler_ext: {}
+    rubygem-clamp: {}
+    rubygem-concurrent-ruby:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-concurrent-ruby-edge:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-css_parser: {}
+    rubygem-daemons: {}
+    rubygem-deacon: {}
+    rubygem-deep_cloneable: {}
+    rubygem-domain_name: {}
+    rubygem-dynflow:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-eventmachine: {}
+    rubygem-excon: {}
+    rubygem-extlib: {}
+    rubygem-facter: {}
+    rubygem-faraday:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
```

Comment:
This is a good red-flag to find! This should not exist in both of these releasers anymore. The `koji-foreman-plugins-nightly` should be dropped since this is in Foreman core repositories now.